CONTRIBUTING.md

```diff
@@ -14,6 +14,12 @@ If you can’t make the change yourself, please open an issue after making sure
 
 Unless you’re specifically improving something about the command-line experience of Carthage, please make code changes to [CarthageKit](README.md#carthagekit). This framework increases modularity, and allows other tools to integrate with Carthage more easily.
 
+## Get started
+
+After checkout, you can run the following command from the cloned directory, and then open the workspace in Xcode:
+
+    ./script/carthage bootstrap --platform mac --no-use-binaries
```

Comment:
As @jspahrsummers pointed out above, this can just be `./script/bootstrap`. :smile: